diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3156a8b..134f615 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -36,9 +36,11 @@ add_definitions(-DN2N_VERSION="${N2N_VERSION}" -DN2N_OSNAME="${N2N_OSNAME}")
 # Build information
 OPTION(BUILD_SHARED_LIBS "BUILD Shared Library" OFF)
 
 # N2n specific params
 OPTION(N2N_OPTION_AES "USE AES" ON)
 
+OPTION(N2N_OPTION_OPENSSL_1_1 "USE OPENSSL1.1" ON)
+
 if(NOT DEFINED N2N_OPTION_AES)
 set(N2N_OPTION_AES ON)
 endif(NOT DEFINED N2N_OPTION_AES)
@@ -127,6 +131,11 @@ endif(N2N_OPTION_AES)
 add_executable(edge src/edge.c)
 target_link_libraries(edge n2n)
 
+if(N2N_OPTION_OPENSSL_1_1)
+  add_definitions(-DHAVE_OPENSSL_1_1)
+  target_link_libraries(edge Crypt32)
+endif(N2N_OPTION_OPENSSL_1_1)
+
 add_executable(supernode src/sn.c)
 target_link_libraries(supernode n2n)
 
