diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8e4ab89..3156a8b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -4,7 +4,7 @@ include(CheckFunctionExists)
 SET(CMAKE_VERBOSE_MAKEFILE ON)
 
 # N2n release information
-set(N2N_VERSION "2.7.0")
+set(N2N_VERSION "2.8.0")
 set(N2N_OSNAME ${CMAKE_SYSTEM})
 execute_process(
         COMMAND git --version
diff --git a/include/n2n.h b/include/n2n.h
index ccabd0a..1b0bb79 100644
--- a/include/n2n.h
+++ b/include/n2n.h
@@ -38,7 +38,7 @@
 #ifdef WIN32
 #include "win32/n2n_win32.h"
 
-#ifdef _MSC_VER
+#ifndef CMAKE_BUILD
 #include "config.h" /* Visual C++ */
 #else
 #include "win32/winconfig.h"
