--- platformio_override.ini
+++ platformio_override.ini
@@ -16,7 +16,7 @@
 ; *** Build/upload environment
 default_envs =
 ; *** Uncomment the line(s) below to select version(s)
-                tasmota
+                tasmota-CN
 ;                tasmota-debug
 ;                tasmota-ircustom
 ;                tasmota-minimal
@@ -38,4 +38,6 @@
 ;                tasmota32solo1
 
+[env:M5_ST]
+build_flags             = ${common.build_flags} -DMY_LANGUAGE=zh_CN -DM5_ST
 
 [common]
