--- a/tasmota/user_config_override.h
+++ b/tasmota/user_config_override.h
@@ -84,8 +84,20 @@
 
 */
 
+#undef  STA_SSID1
+#define STA_SSID1         "OpenWrt_2G"             // [Ssid1] Wifi SSID
 
+#undef  STA_PASS1
+#define STA_PASS1         "13726010595"     // [Password1] Wifi password
 
+#undef  NTP_SERVER1
+#define NTP_SERVER1            "192.168.6.254"       // [NtpServer1] Select first NTP server by name or IP address (129.250.35.250)
+#undef  NTP_SERVER2            // [NtpServer2] Select second NTP server by name or IP address (5.39.184.5)
+#define NTP_SERVER2            "ntp.aliyun.com"
+#undef  NTP_SERVER3            // [NtpServer3] Select third NTP server by name or IP address (93.94.224.67)
+#define NTP_SERVER3            "ntp1.aliyun.com"
+
+#undef  APP_TIMEZONE
+#define APP_TIMEZONE           8                 // [Timezone] +1 hour (Amsterdam) (-13 .. 14 = hours from UTC, 99 = use TIME_DST/TIME_STD)
-

 #endif  // _USER_CONFIG_OVERRIDE_H_
