--- a/tasmota/user_config_override.h
+++ b/tasmota/user_config_override.h
@@ -84,3 +84,16 @@
 
 */
+
+#ifndef USE_TASMOTA_CLIENT
+#define USE_TASMOTA_CLIENT    // Enable the driver
+#endif
+
+#ifndef USE_TASMOTA_CLIENT_FLASH_SPEED
+#define USE_TASMOTA_CLIENT_FLASH_SPEED 57600    // Configure the baud rate of the bootloader
+#endif
+
+#ifndef USE_TASMOTA_CLIENT_SERIAL_SPEED 57600
+#define USE_TASMOTA_CLIENT_SERIAL_SPEED 57600    // Configure the baud rate at which the client microcontroller will be interfacing to Tasmota
+#endif
+
 
